packages/algob/src/lib/lsig.ts

```diff
@@ -17,3 +16,12 @@ export async function getLsig (name: string, scParams: Object, algodClient: algo
   const program = result.toBytes;
   return algosdk.makeLogicSig(program, scParams);
 }
+
+/**
+ * Description: this function creates and returns a dummy logic signature
+ */
+export function getDummyLsig (): LogicSig {
+  const dummyProgram = new Uint8Array(56);
+  dummyProgram.fill(0);
+  return algosdk.makeLogicSig(dummyProgram, []);
+}
```

Comment:
This can be a `const`:
```
export DUMMY_LSIG = algosdk.makeLogicSig(new Uint8Array(56), []);
```

The `  dummyProgram.fill(0);` is not needed, because 0 is a default value already.

Reply:
Okay